lib/veres-one.js

```diff
@@ -0,0 +1,431 @@
+/*!
```

Comment:
We need to provide a `use` API here that wraps injector use, or is that functionality provided elsewhere?

Reply:
I was wondering about that (whether this lib should also provide a `use` API for injector). For the moment, that api only lives in `did-io`. So my initial thought was, it's only there for outside developers, for in-house usage of `did-veres-one`, we can mess with the Injector directly and pass it in.

However, i can totally see how it would be useful. Shall I add it here too?

Reply:
Yes, please.